src/node/constants.ts

```diff
@@ -23,7 +23,24 @@ export const version = pkg.version || "development"
 export const commit = pkg.commit || "development"
 export const rootPath = path.resolve(__dirname, "../..")
 export const vsRootPath = path.join(rootPath, "vendor/modules/code-oss-dev")
+export const codeVersion = require(path.join(vsRootPath, "package.json")).version || "development"
```

Comment:
The e2e tests don't seem to like this. Thoughts/ideas?

```
$ /home/runner/work/code-server/code-server/test/node_modules/.bin/playwright test
Error: Cannot find module '/home/runner/work/code-server/code-server/vendor/modules/code-oss-dev/package.json'
```

Reply:
Hmm... my guess would be that it doesn't like that we're using `require` here and it expects that to be a module? 🤔

That may be what lead me to create [`getPackageJson`](https://github.com/coder/code-server/blob/7d0610320f6aeca26cc591a74315aa099d304dcd/src/node/constants.ts#L8-L17).

There's probably something funky that Playwright does where it requires all the modules before running the tests. And since it can't find a module, it complains. 

Maybe @code-asher knows how to work around this

Reply:
Using `getPackageJson` for `VS Code`'s `package.json` as well makes sense to me.

Maybe we are not packaging the `package.json` as part of the release artifact.

Reply:
> Using getPackageJson for VS Code's package.json as well makes sense to me.

Oh duh...didn't even put two-and-two together. Yeah you could use it here.

Reply:
switched to `getPackageJson`, hopefully that fixes it :crossed_fingers: